src/main/java/uk/gov/hmcts/probate/transformer/CallbackResponseTransformer.java

```diff
@@ -1082,6 +1096,56 @@ private String getSolsSOTName(String firstNames, String surname) {
         return sb.toString();
     }
 
+    private ResponseCaseDataBuilder clearGrantSpecificData(CallbackRequest callbackRequest, ResponseCaseDataBuilder responseCaseDataBuilder, String newState) {
```

Comment:
might be a good idea to move this into its own service, to lessen the burden on the transformer - not essential